scrapers/src/json_loader.ts

```diff
@@ -28,7 +28,7 @@ const getClassMapLinkForTermId = (termId: number) => {
 // ./maps/${TERMID}.json
 const DIR = "maps";
 const getClassMapLinkLocal = (termId: number) => {
-  return `./${DIR}/${termId}.json`;
+  return `../${DIR}/${termId}.json`;
```

Comment:
holy shit is this gonna get used somewhere :eyes: (i wrote it like 3 years ago lol)

Reply:
ahahah na, we re-organized the tests a bit in CI and needed to make this update to make them pass lmao